apps/api/src/app/events/usecases/trigger-event/trigger-event.usecase.ts

```diff
@@ -532,7 +546,7 @@ export class TriggerEvent {
             color: organization.branding?.color || '#f47373',
           },
           blocks: isEditorMode ? content : [],
-          ...command.payload,
+          ...messageVariables,
```

Comment:
```suggestion
          ...command.payload,
          ...subscriber
```

What we want here is to pass the subscriber rather than the payload of the `to` property, since we want to allow be passed as only `id` field rather than an object.